apps/src/templates/sectionProgress/progressTables/ProgressTableLevelBubble.jsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import Radium from 'radium';
+import PropTypes from 'prop-types';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import i18n from '@cdo/locale';
+import * as progressStyles from '@cdo/apps/templates/progress/progressStyles';
+import color from '@cdo/apps/util/color';
+import {LevelStatus} from '@cdo/apps/util/sharedConstants';
+
+const styles = {
+  container: {
+    ...progressStyles.flex,
+    width: progressStyles.BUBBLE_CONTAINER_WIDTH
+  },
+  main: {
+    ...progressStyles.flex,
+    ...progressStyles.font,
+    boxSizing: 'content-box',
+    width: progressStyles.DOT_SIZE,
+    height: progressStyles.DOT_SIZE,
+    borderRadius: progressStyles.DOT_SIZE,
+    fontSize: 16,
+    letterSpacing: -0.11,
+    margin: '3px 0px',
+    position: 'relative'
+  },
+  largeDiamond: {
+    width: progressStyles.DIAMOND_DOT_SIZE,
+    height: progressStyles.DIAMOND_DOT_SIZE,
+    borderRadius: 4,
+    transform: 'rotate(45deg)',
+    margin: '6px 0px'
+  },
+  small: {
```

Comment:
Would calling this `smallCircle` be clearer?  (I see that the intent appears to be that "bubble" implies a circle, but we also call the diamond thing a bubble too, right?)

Reply:
good call, especially since we will eventually need this to support `smallDiamond` as well.